python/cuml/_thirdparty/sklearn/preprocessing/_imputation.py

```diff
@@ -137,7 +141,8 @@ def _more_tags(self):
         return {'allow_nan': is_scalar_nan(self.missing_values)}
```

Comment:
@dantegd This looks like it will be a problem for static tags.

Reply:
Yes, indeed

Reply:
it no longer is a problem and this will work as is :)